lib/parse-args.js

```diff
@@ -155,6 +156,8 @@ function buildYargs (withCommands = false) {
     .pkgConf('c8')
     .demandCommand(1)
     .check((argv) => {
+      // not sure this block is reached
+      /* c8 ignore next 4 */
```

Comment:
While a definite improvement to the coverage score, I think that these comments might not belong in this PR: no adjacent code is being changed, and the addition of the ignore command hides a possible fault. I recommend to remove these two comments, even at the expense of worse code coverage in this PR: better to fix in a dedicated PR where the author and reviewers can go over the details.
(sorry if the above sounds like it was written by a lawyer, just finished watching an episode of JAG...)

Reply:
I agree, let's confirm or deny that this code is necessary.. If we're not hitting the line that sets `/tmp`, what is setting `/tmp`?

Reply:
@bcoe @kf6kjg Just created an issue in the queue.  #489.  Let's move this particular discussion there.